files/content/orbs_for_all/orbs_for_all.lua

```diff
@@ -0,0 +1,57 @@
+-- Put together by ScipioWright
+-- I copied Eba's fuckedupenemies file as a base, so this is me giving credit
+
+local orbs_data = {
+	orb_adjusted = {},
+    enemy_base_health = {},
+}
+
+local function get_scaled_health(health, orb_count)
+    -- thank you Heinermann
+    return health + 2^orb_count + orb_count * health / 3
```

Comment:
```suggestion
    return health + 2^orb_count + orb_count * (health / 3)
```

Reply:
Does lua do division in a weird order or something?

Reply:
it's just order of operations (multiply/divide have same precedence), since we wanted to capture the division first, but I guess it mostly doesn't matter since it's float division (except in the case of precision but there's probably not that much difference in the result anyway).